diff --git a/ArchiSteamFarm/IPC/ArchiKestrel.cs b/ArchiSteamFarm/IPC/ArchiKestrel.cs
index 371d305c..c7f2bdcf 100644
--- a/ArchiSteamFarm/IPC/ArchiKestrel.cs
+++ b/ArchiSteamFarm/IPC/ArchiKestrel.cs
@@ -78,6 +78,7 @@ internal static class ArchiKestrel {
 
 			string customDirectory = Path.Combine(Directory.GetCurrentDirectory(), SharedInfo.WebsiteDirectory);
 			string websiteDirectory = Directory.Exists(customDirectory) ? customDirectory : Path.Combine(AppContext.BaseDirectory, SharedInfo.WebsiteDirectory);
+			ASF.ArchiLogger.LogGenericInfo("website-dir:" + websiteDirectory);
 
 			// Set default content root
 			builder.UseContentRoot(SharedInfo.HomeDirectory);

